<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRxDisplay</class>
 <widget class="QFrame" name="CRxDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>3</number>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="3">
    <widget class="Line" name="line_2">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="recordActivate">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777214</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RecordQso</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="triggerActivate">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>trig activ.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="TriggerText">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CQ CQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FrequencySelect" name="RxFreq">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>106</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Rx Freq. / AFC</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="CSquelch" name="Squelch">
     <property name="minimumSize">
      <size>
       <width>109</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Squelch</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="TabWidget" name="RxHeader">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Light">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>190</red>
             <green>190</green>
             <blue>190</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>239</red>
             <green>235</green>
             <blue>231</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="B_frequency">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>14070000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="B_if_tune">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>IF tune</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="B_tos_meter">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TOS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Clear">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>17</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>CLR</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrequencySelect</class>
   <extends>QGroupBox</extends>
   <header>frequencyselect.h</header>
   <container>1</container>
   <slots>
    <signal>FrequencyChanged(double)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>CSquelch</class>
   <extends>QGroupBox</extends>
   <header>csquelch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>RxFreq</sender>
   <signal>FrequencyChanged(double)</signal>
   <receiver>CRxDisplay</receiver>
   <slot>setRxFrequency(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RxHeader</sender>
   <signal>currentChanged(int)</signal>
   <receiver>CRxDisplay</receiver>
   <slot>changeActiveRxWindow(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Clear</sender>
   <signal>clicked()</signal>
   <receiver>CRxDisplay</receiver>
   <slot>clearRxWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>833</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>triggerActivate</sender>
   <signal>clicked()</signal>
   <receiver>CRxDisplay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TriggerText</sender>
   <signal>returnPressed()</signal>
   <receiver>CRxDisplay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordActivate</sender>
   <signal>toggled(bool)</signal>
   <receiver>CRxDisplay</receiver>
   <slot>record(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setRxFrequency(double)</slot>
  <slot>changeActiveRxWindow(int)</slot>
  <slot>clearRxWindow()</slot>
  <slot>trigger()</slot>
  <slot>record(bool)</slot>
 </slots>
</ui>
